app/uk/gov/hmrc/vatsubscription/config/featureSwitch/FeatureSwitchModel.scala

```diff
@@ -25,4 +25,4 @@ case class FeatureSwitchModel(latestApi1363Version: Boolean,
 
 object FeatureSwitchModel {
   implicit val format: OFormat[FeatureSwitchModel] = Json.format[FeatureSwitchModel]
-}
\ No newline at end of file
+}
```

Comment:
If you've removed the latestApi1363 from config do we still need it here in the Feature Switch Model?